ingestion/src/metadata/ingestion/ometa/mixins/lineage_mixin.py

```diff
@@ -216,6 +216,8 @@ def ingest_lineage_by_query(self, query: str, database: str, service_name: str):
 
         try:
             result = LineageRunner(query)
+            if not result.target_tables:
```

Comment:
Hi, @codingwithabhi could you pls update the signature of the function to mark that it returns a boolean? Also, let's update the description explaining that True means that something got added, and False otherwise.

We could think about refactoring this down the road and return the lineage information or None, but documenting the current behavior should be good enough for now 🙏